editor/components/DancerTree/index.tsx

```diff
@@ -107,6 +107,19 @@ const DancerTree = () => {
     );
   };
 
+  const nodeSortFunction = (a: string, b: string) => {
```

Comment:
Please add some comment for this function.

Reply:
done